webapp/channels/src/components/advanced_create_post/advanced_create_post.test.jsx

```diff
@@ -29,6 +32,10 @@ jest.mock('actions/post_actions', () => ({
     }),
 }));
 
+jest.mock('utils/exec_commands', () => ({
+    execCommandInsertText: jest.fn(),
+}));
```

Comment:
Makes it easy to for the document.execCommand to be testable, which is actually very hard to mock.